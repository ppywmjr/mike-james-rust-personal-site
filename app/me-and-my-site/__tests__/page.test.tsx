import renderer from "react-test-renderer";
import MeAndMySite from "../page";

describe("Me and my site", () => {
  it("renders the page", () => {
    const tree = renderer.create(<MeAndMySite />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
