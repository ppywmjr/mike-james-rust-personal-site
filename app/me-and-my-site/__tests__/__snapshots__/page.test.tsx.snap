// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Me and my site renders the page 1`] = `
<main
  className="main main"
>
  <h1>
    Me and my site
  </h1>
  <p>
    Hi folks, I'm Mike and this is my site. Welcome! This site is primarily for me, but I'm delighted to imagine you reading it. I've made a few little apps just for fun over the years but I'm motivated to make this one for a couple of new reasons. Firstly, I know someone who wants to build a website and they came to me for some advice. I found that as much as I know how to make a huge difference to agile teams, I realised I don't really know how to put together a complete website. So I was a little embarrased. Yes, my core motivation is embarrassment. The second reason is the technology. I've been working as a tester with Nextjs apps for a couple of years now and it's so easy to work with that I figured the effort to get something up and running would be minimal. Also, Nextjs has so many features that I think it would be ready and fun to try out some different approaches on different pages.
  </p>
</main>
`;
